tuned/plugins/plugin_bootloader.py

```diff
@@ -355,20 +357,46 @@ def _cmdline(self, enabling, value, verify, ignore_missing):
 				return None
 			cmdline_set = set(cmdline.split())
 			value_set = set(v.split())
-			cmdline_intersect = cmdline_set.intersection(value_set)
-			if cmdline_intersect == value_set:
+			missing_set = value_set - cmdline_set
+			if len(missing_set) == 0:
 				log.info(consts.STR_VERIFY_PROFILE_VALUE_OK % ("cmdline", str(value_set)))
 				return True
 			else:
-				log.error(consts.STR_VERIFY_PROFILE_VALUE_FAIL % ("cmdline", str(cmdline_intersect), str(value_set)))
+				cmdline_dict = {v.split("=", 1)[0]: v for v in cmdline_set}
+				for m in missing_set:
+					arg = m.split("=", 1)[0]
+					if not arg in cmdline_dict:
+						log.error(consts.STR_VERIFY_PROFILE_VALUE_FAIL % ("cmdline arg "+arg, "<missing>", m))
+					else:
+						log.error(consts.STR_VERIFY_PROFILE_VALUE_FAIL % ("cmdline arg "+arg, cmdline_dict[arg], m))
+				present_set = value_set & cmdline_set
+				log.info("expected arguments that are present in cmdline: %s"%(" ".join(present_set),))
 				return False
 		if enabling and value is not None:
 			log.info("installing additional boot command line parameters to grub2")
 			self.update_grub2_cfg = True
 			self._cmdline_val = v
 
+	@command_custom("skip_grub_config", per_device = False, priority = 10)
+	def _skip_grub_config(self, enabling, value, verify, ignore_missing):
+		if verify:
+			return None
+		if enabling and value is not None:
+			if value.lower() in "yes true t 1".split():
+				log.info("skipping any modification of grub config")
+				self._skip_grub_config_val = True
+
 	def _instance_post_static(self, instance, enabling):
-		if enabling and self.update_grub2_cfg:
+		if enabling and self._skip_grub_config_val:
```

Comment:
Lines 390 - 399: do we need it? I.e. with the `skip_grub_config = True` should we care what's already there? There is 'verification' for the check. Or if we care, shouldn't we throw exception (e.g. InvalidProfileException) to stop loading of the profile? I am just thinking, this comment is not request for change.

Reply:
Yes, I was wondering the same. I don't think profile loading should fail because of a wrong cmdline. But I would be ok with not checking during profile load, and require an explicit `tuned-adm verify` instead.

Reply:
I don't have strong opinion on it, I think we could merge it as it is. @CZerta what's your opinion?

Reply:
I don't have strong opinion too, but probably wouldn't verify it either. Without verification it can theoretically be used in example on systems without "/proc/cmdline" file and won't log error.

Reply:
I have removed the verification during profile loading.